src/components/Book.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/forbid-prop-types */
 import PropTypes from 'prop-types';
```

Comment:
- [ ] <b>REQUIRED:</b> There is no need to use `eslint-disable` here. besides, you are not allowed to use it unless it is totally obvious that there is no other choice. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). to fix this, replace line 21 with this:
```Javascript
 item: PropTypes.shape({}).isRequired,
```

Reply:
Thanks a lot, done.

Reply:
I fixed it but other problems arose:
```
  5:15  error  'item.id' is missing in props validation        react/prop-types
  6:15  error  'item.title' is missing in props validation     react/prop-types
  7:15  error  'item.category' is missing in props validation  react/prop-types
```
Can you tell me how to fix them?